client/src/app/infrastructure/utils/subscription-map.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Observable, Subscription } from 'rxjs';
+import { SubscriptionMap } from 'src/app/infrastructure/utils/subscription-map';
+
+const sub = new SubscriptionMap();
+
+describe(`utils: subscription-map`, () => {
+    describe(`delete method`, () => {
+        it(`delete one subscription from two`, () => {
+            let subscription: Subscription = new Observable().subscribe();
+            let subscription2: Subscription = new Observable().subscribe();
+            sub.updateSubscription(`test`, subscription);
+            sub.updateSubscription(`test2`, subscription2);
+            sub.delete(`test`);
+            expect(subscription.closed).toBe(true);
+            expect(subscription2.closed).toBe(false);
+        });
+    });
```

Comment:
Remove the `describe` around the `it` blocks.

```suggestion
        it(`delete one subscription from two`, () => {
            let subscription: Subscription = new Observable().subscribe();
            let subscription2: Subscription = new Observable().subscribe();
            sub.updateSubscription(`test`, subscription);
            sub.updateSubscription(`test2`, subscription2);
            sub.delete(`test`);
            expect(subscription.closed).toBe(true);
            expect(subscription2.closed).toBe(false);
        });
```